automated install/basic-install.sh

```diff
@@ -145,6 +145,7 @@ EOM
 # The runUnattended flag is one example of this
 reconfigure=false
 runUnattended=false
+PIHOLE_SKIP_FTL_CHECK=false
```

Comment:
How can it be set from the outside when it is overwritten here?

Reply:
A check was added